db/migrate/20210604184858_add_genre_reference_to_movie.rb

```diff
@@ -0,0 +1,8 @@
+class AddGenreReferenceToMovie < ActiveRecord::Migration[6.1]
+  def change
+    change_table :movies do |t|
+      t.remove :genre, type: :string
+      t.belongs_to :movie_genre
```

Comment:
```suggestion
      t.belongs_to :movie_genre
```
This means that movie_genre_id could be nullable in the `movies` table. Is OK?

Reply:
It wasn't intended - I didn't think about it 🙈 Thank you! I changed that 👌